text_reading/src/main/scala/org/clulab/aske/automates/scienceparse/ScienceParseClient.scala

```diff
@@ -58,8 +58,8 @@ object ScienceParseClient {
     val authors = json("authors").arr.map(_.str).toVector
     //val venue = json("venue").str
     val venueOption = json.obj.get("venue").map(_.str)
-    val year = json("year").num.toInt
```

Comment:
Science parse didn't always get the year for some reason.

Reply:
cool -- we can make it an option if we want it, up to you

Reply:
👍